main.py

```diff
@@ -0,0 +1,423 @@
+from fastapi import FastAPI, HTTPException, Path, Body
+from fastapi.encoders import jsonable_encoder
+from typing import Annotated
+from models.Dispatcher import Dispatcher, UpdateDispatcherModel
+from models.Repair import RepairModel
+from models.Trailer import Trailer, UpdateTrailerModel
+from models.Truck import Truck, UpdateTruckModel
+from models.Manager import Manager, UpdateManagerModel
+from get_db_pymongo import get_database
+from bson import ObjectId
+from models.TruckDriver import TruckDriver, UpdateTruckDriverModel
+from utils import serialize_collection
+
+app = FastAPI()
+
+db = get_database()
```

Comment:
Instead of getting db. Can't we create a function that does all for us?

For example:

def get_trucks_collection():
db = get_database()
return db.get_collection("trucks")

And we should do this for every collection